scripts/01_prep_inputs/0100-extract-inputs.R

```diff
@@ -802,22 +795,22 @@ hab_fish_input_prep2 <- left_join(
   #janitor::adorn_totals()   ##use this to ensure you have the same number of fish in the summary as the individual fish sheet
 
 
-## Read in form_fiss to get the following fields: Temp, conductivity, turbidity, and site length.
+## Read in form_fiss to get the site length.
 form_fiss_site_raw <- sf::st_read(dsn= paste0('../../gis/', dir_gis, '/data_field/2023/form_fiss_site_', year, '.gpkg')) |>
   st_drop_geometry()
 
 
-## Read in step_4_stream_site_data to get the average wetted width which is the site width
+## Read in step_4_stream_site_data to get the average wetted width which is the site width, temp, conductivity, and turbidity.
 hab_site_dat_raw <- habitat_confirmations |>
   purrr::pluck("step_4_stream_site_data") |>
   dplyr::filter(!is.na(site_number))
 
 
 ## Join columns together so we can add them to the fish data (hab_fish_input_prep2)
 hab_site_dat <- left_join(form_fiss_site_raw |>
-                            select(local_name, temperature_c, conductivity_m_s_cm, turbidity, site_length),
+                            select(local_name, site_length),
                           hab_site_dat_raw |>
-                            select(reference_number, local_name, avg_wetted_width_m) |>
+                            select(reference_number, local_name, temperature_c, conductivity_m_s_cm, turbidity, avg_wetted_width_m) |>
```

Comment:
hmm.  a bit wary of this without any note to the user `since dplyr::fill_down` (or up or whatever) will save us dozens and dozens of hand inputs in the future (its easy and a common move to share data between sites with a common basename - ex. if `1234_us` has the wq data - it can be the parent for all wq data for all other 7 `1234_*` sites assessed/sampled that day).

in the future (not in this file now pretty please) we should alert the user of this sort of change in a comment with a note that the change is the result of a non preferred workflow and will need to be reversed in the future

Reply:
oops sorry was planning on putting a note in there about `fill_down` but forgot. Will make a note for the future. Thanks